components/external-link/index.js

```diff
@@ -1,40 +0,0 @@
-/**
- * External dependencies
- */
-import classnames from 'classnames';
-import { compact, uniq } from 'lodash';
-
-/**
- * WordPress dependencies
- */
-import { __ } from '@wordpress/i18n';
-
-/**
- * Internal dependencies
- */
-import Dashicon from '../dashicon';
-import './style.scss';
-
-function ExternalLink( { href, children, className, rel = '', ...additionalProps } ) {
```

Comment:
We should probably leave this for two versions per deprecation policy, with an added `deprecated` warning.

https://wordpress.org/gutenberg/handbook/reference/deprecated/
https://github.com/WordPress/gutenberg/blob/master/utils/deprecation.js